src/xbot/common/controls/actuators/XCANTalon.java

```diff
@@ -185,6 +185,10 @@ public void ensureTalonControlMode(TalonControlMode mode) {
     public abstract void configForwardLimitSwitchNormallyOpen(boolean normallyOpen);
     public abstract void configReverseLimitSwitchNormallyOpen(boolean normallyOpen);
     
+    // Current --------------------------------------------
+    public abstract void enableCurrentLimit(boolean enable);
+    public abstract void setCurrentLimit(int amps);
```

Comment:
Is this an integer all the way through?

Reply:
In the current CTRE libraries, their API only exposes an integer. It may be that under the hood they change the value - I haven't looked that deep.

I think we can go with this for now, and review their source/api next year when we review their new libraries.